R/zzz.R

```diff
@@ -107,3 +122,10 @@ global_variables(c("name", "parent", "properties", "constructor", "validator"))
   env <- as.environment(paste0("package:", pkgname))
   env[[".conflicts.OK"]] <- TRUE
 }
+
+.onLoad <- function(...) {
+  base_classes$`NULL` <- new_base_class("NULL")
```

Comment:
```suggestion
  base_classes$`NULL` <<- new_base_class("NULL")
```

? just guessing based on the others